### ISSUE TITLE
A11y_Visual Studio Code Client_Installation_Screen Reader: Jaws is not announcing any status message about the installation progress.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode#203296]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG4.1.3; #Status Message; #JAWS; #NVDA; #A11yMAS;

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007

### Repro Steps:

1. Open URL ' https://code.visualstudio.com/insiders/'
2. Download Visual Studio Code Insiders setup for Windows and save it.
3. Double click on the "vscodesetup-stable.exe" to start installation.
4. Select "Run" to start installation process.
5. Installation window will appear TAB till "Next" button and press ENTER.    
6. Accept the agreement and TAB till "Next" button and press ENTER.       
7. Select any folder if you want else TAB till "Next" button and press ENTER.          
8. Select any folder if you want else TAB till "Next" button and press ENTER.                                                                         
9. Select the additional task you want to setup for installation and TAB till "Next" button and press ENTER.      
10. Start JAWS and TAB till "Install" button and press ENTER.     
11. Observe that Jaws is not announcing any status message about the installation progress.

### Actual Result:
Jaws is not announcing any status message about the installation progress.

**This issue is not observed with NVDA.**

### Expected Result:
Jaws should announce status messages appearing on to the screen about the installation progress.

### User Impact:
Screen reader only users will not be able to get any information about the installation progress. 

Attachment:
![Jaws is not announcing any status message about the installation progress](https://github.com/microsoft/vscode/assets/115216656/65cfa20d-3e97-4108-a7bd-8c25cada33cb)
![Screenshot 2024-01-24 145341](https://github.com/microsoft/vscode/assets/115216656/965e3f0f-582c-4a4b-a50b-9a2ea9f04190)

**Comment by meganrogge** [comment]
Given that this works with NVDA, closing as this is an issue with JAWS. Please create an issue for them.
